### ISSUE TITLE
"clear selection" and "more" in selection toolbar is not accessible through keyboard navigation

### ISSUE THREAD

**Issue by pernyl** [qlik-oss/sn-list-objects#275]
Try to trigger clear selection or more ithe keyboard navigation
